api/src/main/java/org/pmiops/workbench/api/NotebooksController.java

```diff
@@ -200,24 +196,4 @@ public ResponseEntity<NotebookLockingMetadataResponse> getNotebookLockingMetadat
 
     return ResponseEntity.ok(response);
   }
-
-  private String findHashedUser(String bucket, Set<String> workspaceUsers, String hash) {
```

Comment:
these 2 methods were duplicated between here and CloudStorageClient - now moved to a new NotebookLockingUtils